conviva/src/main/java/com/bitmovin/analytics/conviva/ConvivaAnalyticsIntegration.java

```diff
@@ -468,22 +472,24 @@ private synchronized void transitionState(ConvivaSdkConstants.PlayerState state)
         @Override
         public void onEvent(PlayerEvent.Error event) {
             Log.d(TAG, "[Player Event] Error");
-            String message = String.format("%s - %s", event.getCode(), event.getMessage());
-            convivaVideoAnalytics.reportPlaybackError(message, ConvivaSdkConstants.ErrorSeverity.FATAL);
-            internalEndSession();
+            handleError(String.format("%s - %s", event.getCode(), event.getMessage()));
         }
     };
 
     private final EventListener<SourceEvent.Error> onSourceErrorListener = new EventListener<SourceEvent.Error>() {
         @Override
         public void onEvent(SourceEvent.Error event) {
             Log.d(TAG, "[Source Event] Error");
-            String message = String.format("%s - %s", event.getCode(), event.getMessage());
-            convivaVideoAnalytics.reportPlaybackError(message, ConvivaSdkConstants.ErrorSeverity.FATAL);
-            internalEndSession();
+            handleError(String.format("%s - %s", event.getCode(), event.getMessage()));
```

Comment:
Would it make sense to make the message creation also reusable? i.e. wrap or move the string format stuff also into the `handleError` function?